workflow/github.md

```diff
@@ -82,7 +82,7 @@ Here are the steps that need to be completed before merging:
 - Your pull request must be approved by at least one person
 - Any automated checks must have passed
 - The original author should merge a pull request
-  - This can be overruled when necessary, only if the author is unavailable
+  - This can be overruled when necessary, f.e. if all looks good and the changes are needed for further development and the author is AFK.
```

Comment:
f.e. ? Do you mean e.g. ?

Reply:
I'm cool with this change. But I agree you should change it to either "e.g.", or "for example"

Reply:
updated :)